must-gather/node-gather/clusterrole.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: perf-node-gather-pods-reader
+rules:
+- apiGroups: [""]
+  resources: ["pods"]
+  verbs: ["get","list"]
+  
```

Comment:
is it asking for an additional new line or to remove the empty line?

Reply:
remove empty line probably.  compare the format with other yamls in the folder